src/main/java/com/sparta/vicky/comment/entity/Comment.java

```diff
@@ -0,0 +1,83 @@
+package com.sparta.vicky.comment.entity;
+
+import com.sparta.vicky.baseEntity.Timestamped;
+import com.sparta.vicky.comment.dto.CommentRequest;
+import com.sparta.vicky.user.entity.User;
+import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Entity
+@Getter
+@NoArgsConstructor
```

Comment:
엔티티의 기본 생성자는 다음과 같이 걸어주세요.
```suggestion
@NoArgsConstructor(access = AccessLevel.PROTECTED)
```